src/features/presence/utils/presenceUtils.js

```diff
@@ -60,4 +64,24 @@ function determineInteractionMessage(interactionType, interactedAt) {
     return message;
 }
 
-export { sortByActivity, getLastActionTimeMS, determineInteractionMessage };
+function convertMillisecondsToUnitAndValue(ms) {
+    const seconds = ms / 1000;
+    const abs = Math.abs(seconds);
+    if (abs < SEC_IN_MINUTE) {
+        return { unit: 'second', value: seconds };
+    }
+    if (abs < SEC_IN_HOUR) {
+        return { unit: 'minute', value: Math.round(seconds / SEC_IN_MINUTE) };
+    }
+    if (abs < SEC_IN_DAY) {
+        return { unit: 'hour', value: Math.round(seconds / SEC_IN_HOUR) };
+    }
+
+    if (abs < SEC_IN_MONTH) {
+        return { unit: 'day', value: Math.round(seconds / SEC_IN_DAY) };
+    }
+
```

Comment:
super small nit:
could we make the white space around this `if` block be consistent the other `if` blocks in this function?